### PR TITLE
Auth redux

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
This whole app was setup you can check this whole app
Thanks for reviewing

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,12 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+      <!-- Fonts awesome linked -->
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
```
Comment (Hamza-777) [inline public/index.html:28]: Try using react-icons package, it is much better than font-awesome.

File: src/Components/Authentication/LogInForm.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+function LogInForm({children}) {
+  const [viewPassword, setViewPassword] = useState(false);
+  const [details, setDetails] = useState({
+    userName: "",
+    password: "",
+  });
+  const dispatch = useDispatch();
+  function clickHandler(e) {
+    //  to prevent initial refreshing of the page
+    e.preventDefault(details);
```
Comment (Hamza-777) [inline src/Components/Authentication/LogInForm.jsx:14]: What does passing in details do here?

**Review by Hamza-777 — APPROVED** [review]
Nice job.

File: src/Components/Footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import { LinkedIn } from "Assets/Icons/LinkedIn";
+import { Twitter } from "Assets/Icons/Twitter";
+function Footer() {
+  return (
+    <footer className="text-gray-600 body-font">
+      <div className="container px-5 py-8 mx-auto flex items-center sm:flex-row flex-col">
+        <a className="flex title-font font-medium items-center md:justify-start justify-center text-gray-900">
+       
+          <img
+            src={require("./../../Assets/Images/FooterImage.png")}
```
Comment (nayyyhaa) [inline src/Components/Footer/Footer.jsx:12]: suggestion: you can first import the img then use it for readability and reusablity.

**Review by nayyyhaa — APPROVED** [review]
Code is readable & easily understandable. +1 to the points mentioned by Hamza above. Rest code looks good to merge. 👍

File: public/index.html (hunk 1 above)
Comment (Dev-AkshatGupta) [reply public/index.html:28]: I am with both bhai Thanks for reviewing

File: src/Components/Authentication/LogInForm.jsx (hunk 2 above)
Comment (Dev-AkshatGupta) [reply src/Components/Authentication/LogInForm.jsx:14]: Removed it it was a mistake I made

File: src/Components/Footer/Footer.jsx (hunk 3 above)
Comment (Dev-AkshatGupta) [reply src/Components/Footer/Footer.jsx:12]: Yeah thanks for suggestion  and reviewing the code

**Pull request merged**